server/src/main/java/org/apache/iotdb/db/mpp/plan/analyze/AnalyzeVisitor.java

```diff
@@ -243,7 +243,7 @@ public Analysis visitQuery(QueryStatement queryStatement, MPPQueryContext contex
       if (queryStatement.isGroupByTag()) {
         schemaTree = schemaFetcher.fetchSchemaWithTags(patternTree);
       } else {
-        schemaTree = schemaFetcher.fetchSchema(patternTree);
+        schemaTree = schemaFetcher.fetchSchema(patternTree, context);
       }
```

Comment:
what is the `context` usage? the same as blow

Reply:
The context contains information about the session

Reply:
However, I do not see it's usage.